Packages/Sites/Neos.NeosIo/package.json

```diff
@@ -10,7 +10,7 @@
     "build:scripts": "browserify -t [ babelify ] -p browserify-derequire Resources/Private/JavaScript/index.js > Resources/Private/JavaScript/Bundle.min.js",
     "fix:amd": "fandr Resources/Public/JavaScript/Bundle.min.js '{\"&& define.amd\": \"&& false\"}'",
     "build:sass": "node-sass Resources/Private/Scss/ -o Resources/Public/Styles/",
-    "postbuild:sass": "postcss --use autoprefixer Resources/Public/Styles/*.css -d Resources/Public/Styles/",
```

Comment:
Personally I think having tasks in the package file is great, but it's really hard to read and maintain with the longer commands.
Grunt/Gulp deliver a bit more clarity there in my opinion. But we can discuss this :)